### ISSUE TITLE
Facing issue in importing nextjs

### ISSUE THREAD

**Issue by avneet2112** [alextanhongpin/compress.js#13]
When I try to use compress.js latest version i.e. 1.2.1 then it shows a build error that regenerator runtime cannot import from outside the module. Please look into it.

**Comment by maitrungduc1410** [comment]
same issue here. here's detail:

```
Module parse failed: 'import' and 'export' may appear only with 'sourceType: module' (2:0)
File was processed with these loaders:
 * ./node_modules/next-translate/lib/cjs/plugin/loader.js
You may need an additional loader to handle the result of these loaders.
| // Support regenerator-runtime globally.
> import "regenerator-runtime/runtime.js";
| 
| import Base64 from "./core/base64.js";
```

**Comment by nemanjam** [comment]
I get same build error in Next.js app.

**Comment by bruceharrison1984** [comment]
I just got this as well in NextJS

**Comment by avneet2112** [comment]
I also got a solution here.

You can restrict the compress.js to the last update where it has 0 dependencies.
inside your package.json
replace this
"compress.js": "^1.1.2",
with this
"compress.js": "1.1.2",

**Comment by alextanhongpin** [comment]
Should be fixed in 1.2.2
